modules/item/sheets/base.js

```diff
@@ -1,36 +1,50 @@
-export default class AKRPGItemSheet extends ItemSheet {
-    constructor(...args) {
-        super(...args);
-    }
-
-    /** @override */
-    static get defaultOptions() {
-        return mergeObject(super.defaultOptions, {
-            resizable: true,
-            scrollY: [".tab.details"],
-            tabs: [
-                {
-                    navSelector: ".tabs",
-                    contentSelector: ".sheet-body",
-                    initial: "description",
-                },
-            ],
-        });
-    }
-
-    /** @override */
-    get template() {
-        const path = "systems/akrpg/templates/item/";
-        return `${path}${this.item.data.type}-sheet.html`;
-    }
-
-    /** @override */
-    getData() {
-        const data = super.getData();
-        console.log("Item Data", data);
-
-        data.itemType = data.item.type.toLowerCase();
-
-        return data;
-    }
-}
+import { AKRPG } from "../../config.js";
+
+export default class AKRPGItemSheet extends ItemSheet {
+    constructor(...args) {
+        super(...args);
+    }
+
+    /** @override */
+    static get defaultOptions() {
+        return mergeObject(super.defaultOptions, {
+            resizable: true,
+            scrollY: [".tab.details"],
+            tabs: [
+                {
+                    navSelector: ".tabs",
+                    contentSelector: ".sheet-body",
+                    initial: "description",
+                },
+            ],
+        });
+    }
+
+    /** @override */
+    get template() {
+        return `systems/akrpg/templates/item/${this.item.data.type}-sheet.html`;
+    }
+
+    /** @override */
+    getData() {
+        const data = {
+          ...super.getData(),
+          config: AKRPG,
+          labels: {},
+        }
+        console.log("Item Data", data);
+
+        data.itemType = data.item.type.toLowerCase();
+
+        return data;
+    }
+
+    /**
+     * @private
+     * Rolls spell damage when called
+     */
+    _rollDamage(event) {
```

Comment:
Can we qualify the name as `_rollSpellDamage()`? I'm not entirely sure why this function exists in the `base.js` if its purpose is solely tied to spells.